### ISSUE TITLE
Remove duplicate social media tags

### ISSUE THREAD

**Issue by steffenkleinle** [digitalfabrik/integreat-app#2823]
**Is your feature request related to a problem? Please describe.**
We implemented frontend side social media tags a while ago to allow for better support in social media.
However, this led to problems since the tags were dynamically rendered and not present on initial load.
Therefore, we decided to move this to the backend, which was completed in https://github.com/digitalfabrik/integreat-cms/pull/2391.
Therefore we can remove the corresponding tags in the frontend.

**Describe the solution you'd like**
Remove all duplicated tags and logic from the frontend (web).

**Describe alternatives you've considered**
None.

**Additional context**
https://github.com/digitalfabrik/integreat-cms/pull/2391

**Comment by LeandraH** [comment]
We currently seem to not have previews when sharing on Telegram or Signal. We probably shouldn't touch this until the preview works again.

**Comment by steffenkleinle** [comment]
> We currently seem to not have previews when sharing on Telegram or Signal. We probably shouldn't touch this until the preview works 

Thank you for investigating. Can you elaborate this a little more please? Are previews for Telegram/Signal working atm and would stop working if we'd remove this? Or are the previews not working in any case?

Because if they are not working in any case, removing it now should not make a difference, right?

**Comment by LeandraH** [comment]
> > We currently seem to not have previews when sharing on Telegram or Signal. We probably shouldn't touch this until the preview works
> 
> Thank you for investigating. Can you elaborate this a little more please? Are previews for Telegram/Signal working atm and would stop working if we'd remove this? Or are the previews not working in any case?
> 
> Because if they are not working in any case, removing it now should not make a difference, right?

No, we currently don't have the previews for Telegram or Signal. It ideally shouldn't make a difference if we remove stuff now or when the previews work but if something goes wrong, I believe it will be easier to fix if we only have one codebase to look in. Plus this isn't exactly urgent, so I'd prefer waiting. What do you think?

**Comment by steffenkleinle** [comment]
> > > We currently seem to not have previews when sharing on Telegram or Signal. We probably shouldn't touch this until the preview works
> > 
> > 
> > Thank you for investigating. Can you elaborate this a little more please? Are previews for Telegram/Signal working atm and would stop working if we'd remove this? Or are the previews not working in any case?
> > Because if they are not working in any case, removing it now should not make a difference, right?
> 
> No, we currently don't have the previews for Telegram or Signal. It ideally shouldn't make a difference if we remove stuff now or when the previews work

Agreed.

> but if something goes wrong I believe it will be easier to fix if we only have one codebase to look in.

I don't really get that. What do you mean by `only one codebase`? The CMS already implemented all the preview stuff and it is shipped/live. Therefore the things we do in the frontend are probably overwritten and adjusted, so like this we have more stuff to check why things are the way they are. If we remove this, there is only a single source of truth (i.e. the CMS).
What am I missing? :see_no_evil: 

> Plus this isn't exactly urgent, so I'd prefer waiting. What do you think?

True, its not urgent :)

**Comment by LeandraH** [comment]
> > but if something goes wrong I believe it will be easier to fix if we only have one codebase to look in.
> 
> I don't really get that. What do you mean by `only one codebase`? The CMS already implemented all the preview stuff and it is shipped/live. Therefore the things we do in the frontend are overwritten and adjusted anyway, so like this we have more stuff to check why things are the way they are. If we remove this, there is only a single source of truth (i.e. the CMS). What am I missing? 🙈

Well, it is shipped but currently not working, which might just be because of the missing Salt PR, but it might not. So I'd rather not change around things at our end too, because if things then don't work, it might be a problem in the Salt, or in the CMS, or in the App. Currently, if things don't work, it might be a problem in the Salt or the CMS since they did somewhat work at some point in the App. Actually, I'm not sure, did the previews on Telegram or Signal ever work?

**Comment by steffenkleinle** [comment]
> > > but if something goes wrong I believe it will be easier to fix if we only have one codebase to look in.
> > 
> > 
> > I don't really get that. What do you mean by `only one codebase`? The CMS already implemented all the preview stuff and it is shipped/live. Therefore the things we do in the frontend are overwritten and adjusted anyway, so like this we have more stuff to check why things are the way they are. If we remove this, there is only a single source of truth (i.e. the CMS). What am I missing? 🙈
> 
> Well, it is shipped but currently not working, which might just be because of the missing Salt PR, but it might not. So I'd rather not change around things at our end too, because if things then don't work, it might be a problem in the Salt, or in the CMS, or in the App. Currently, if things don't work, it might be a problem in the Salt or the CMS since they did somewhat work at some point in the App. Actually, I'm not sure, did the previews on Telegram or Signal ever work?

I actually don't think they every worked in the app. I really still don't understand the point :see_no_evil: If it works after the salt PR, we then remove this stuff and then perhaps it doesn't work we have to investigate again?

**Comment by LeandraH** [comment]
You know, go for it :)

**Comment by bahaaTuffaha** [comment]
What is the salt PR?😅

**Comment by LeandraH** [comment]
Ah yes, good question. Our infrastructure is managed on Gitea, and there is an open PR on our Saltstack there: https://git.tuerantuer.org/DF/salt/pulls/171 (If it says there's nothing to see there, try logging in and then opening it again. If you still don't have access to it, ask Rene about it, or Sven when he's back from vacation next week)

**Comment by svenseeberg** [comment]
Additional info can also be found in https://tasks.tuerantuer.org/projects/infra/work_packages/3734/activity?query_id=59

**Comment by svenseeberg** [comment]
It seems that the HTML structure for social media bots on integreat.app is totally broken:
```
 % curl --user-agent "Mattermost-Bot" https://webnext.integreat.app/augsburg/de/willkommen/willkommen-in-augsburg-2
 <!doctype html><meta charset="utf-8"><meta http-equiv="x-ua-compatible" content="ie=edge"><meta name="viewport" content="width=device-width,initial-scale=1">[...]
 ```

**Comment by svenseeberg** [comment]
> We implemented frontend side social media tags a while ago to allow for better support in social media.

IMHO duplicate(!) social media / preview relevant tags should not be removed but included in the "else" part of the Nginx server side includes. This ensures that we have previews when, for example, the bot is not recognized by our regex expression in https://git.tuerantuer.org/DF/salt/src/branch/master/states/integreat-webapp/files/nginx.conf#L13

**Comment by svenseeberg** [comment]
Duplicate tags:
```
<meta property="og:image" content="/social-media-preview.png" data-react-helmet="true"/>
<meta property="og:image:width" content="1200" data-react-helmet="true"/>
<meta property="og:image:height" content="630" data-react-helmet="true"/>
```
